avr/bootloaders/HoodLoader2/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+cat vidpid.txt | while read mcu vid pid name; do
+    make clean
+    make VID=$vid PID=$pid MCU=$mcu NAME="$name" "$@" all
+    # Please tell me where to find thebuild products...
```

Comment:
That is much more complicated. This build script, being a Shell script, expects the Make command have a consistent output file name.